topology/probes/netlink.go

```diff
@@ -79,6 +79,26 @@ type NetLinkProbe struct {
 	wg      sync.WaitGroup
 }
 
+// RouteTable describes a list of Routes
+type RoutingTable struct {
+	Id      int
+	IfIndex int
```

Comment:
not sure we need to keep this as we attach the RT to the node which already as a IfIndex ?

Reply:
agree and removed